content/en/vendor/README.md

```diff
@@ -0,0 +1,160 @@
+
+    {{< /tabs >}}
+    ```
+1.  If it's not already there, add the vendor disclaimer at the top of the
+    document, below the front matter.
+
+    ```html
+    {{% pageinfo %}}
+    {{% readfile "/vendor/disclaimer.md" %}}
+    {{% /pageinfo %}}
+
+    ```
+
+## Publishing new guide
+
+If you want to publish a standalone document instead of contributing to an
+existing one:
+
+1. Write your new document and save it under
+   `content/en/vendor/<your-vendor>/`. Do not use a first level header for the
+   title. You are going to set the title of the document in a subsequent step.
+
+1. Determine the correct place show the document, that would usually be in one
+   one of the following sections:
+
+    - Installation
+    - Getting Started
+    - How-to Guides
+
+   See the [content guidelines][content-guidelines] for more information.
+
+1. In the folder corresponding to the section you selected, create a new
+   file with a descriptive name. Add a [hugo front matter][hugo-frontmatter] in YAML
+   format. For example, suppose you are adding a new installation guide, create
+   the file `content/en/docs/Installation/my-vendor-install.md` withe following
+   front matter:
+
+    ```yaml
+    ---
+    title: Install Tekton on my cloud service
+    linkTitle: Install Tekton on my cloud service
+    weight: 3
+    description: >
+      Installation guide to run Tekton on my cloud service.
+    ---
+    ```
+
+   Keep in mind that the `weight` parameter determines the order in the left
+   navigation pane. Lower appears higher, documents with the same weight are
+   ordered alphabetically.
+
+1.  Add the vendor disclaimer at the top of the document, below the front
+    matter.
+
+    ```html
+    {{% pageinfo %}}
+    {{% readfile "/vendor/disclaimer.md" %}}
+    {{% /pageinfo %}}
+
+    ```
+
+1. Import your file with the `readfile` shortcode. The final markdown file looks
+   like this:
+
+   ```md
+    ---
+    title: Install Tekton on my cloud service
+    linkTitle: Install Tekton on my cloud service
+    weight: 3
+    description: >
+      Installation guide to run Tekton on my cloud service.
+    ---
+
+    {{% pageinfo %}}
+    {{% readfile "/vendor/disclaimer.md" %}}
+    {{% /pageinfo %}}
+
+    {{% readfile "/vendor/my-vendor/installation.md" %}}
+
+   ```
+
+## Caveats
+
++  Because Docsy, the platform Tekton docs are built on, rebuilds files when
+   their content changes, it will not pick up the changes in your included files
+   unless you make a change on the parent file. You can make a small change,
+   like increasing a dummy counter, while writing content to trigger rebuilding
+   the parent file.
+
++  Documentation using this approach is meant to be read on the website. Reading
+   the source files on GitHub will not be the best experience.
+
+## Own your docs
+
+If contribute documentation to show how Tekton works with a particular platform,
+we encourage you to own your documentation.
+
+1. Ensure freshness and accuracy.
```

Comment:
Should this be an unordered list or are we implying relative importance of these?

Reply:
Good point! updated